rskj-core/src/test/java/co/rsk/peg/BridgeSupportProcessFundsMigrationTest.java

```diff
@@ -241,6 +283,12 @@ private void test_processFundsMigration(
                 entry.getTransaction(),
                 Coin.COIN
             );
+
+            if (activations.isActive(ConsensusRule.RSKIP376)){
+                Assertions.assertEquals(BRIDGE_BTC_TX_VERSION_2, entry.getTransaction().getVersion());
+            } else {
+                Assertions.assertEquals(BRIDGE_BTC_TX_LEGACY_VERSION, entry.getTransaction().getVersion());
+            }
```

Comment:
```suggestion
            int expectedVersionNumber = activations.isActive(ConsensusRule.RSKIP376) ? 2 : 1;
                Assertions.assertEquals(expectedVersionNumber, entry.getTransaction().getVersion());
```
Just an idea to make it shorter, if you like it

Reply:
I like it, but using constants instead of literals(see my comment above). What do you think?